api-documentation.md

```diff
@@ -78,7 +78,7 @@ Note that the shaared URL must not exists. If none is provided, a note will be c
 ## Link [/links/{linkID}]
 
 + Parameters
-    + linkID: `20160606_101010` (string, required) - Link identifier
+    + linkID: `349` (number, required) - Link identifier
```

Comment:
integer isn't defined in API Blueprint. [ref](https://apiblueprint.org/documentation/mson/specification.html#211-primitive-types)